com.hazelcast.mapstore.mysql.MySQLGenericMapStoreIntegrationTest.testDestroy

_master_ (commit 66094111c244d948f883e9b5dd5ae56bae082c1b)

Failed on oracle-8 (little-endian): https://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-little-endian/242/testReport/com.hazelcast.mapstore.mysql/MySQLGenericMapStoreIntegrationTest/testDestroy/
<details><summary>Stacktrace:</summary>

```
java.lang.AssertionError: 

Expecting
  [Row{[__map-store.table_o_096cc44c_fb22_42b1_a80b_e864148776a0]},
    Row{[__map-store.table_o_68d997fe_d546_45fb_8651_8a2b08315b5f]},
    Row{[__map-store.table_o_741c4c93_cdc2_49ae_ba87_d500d3945898]},
    Row{[__map-store.table_o_94f6e2d4_1b39_4f98_8d4c_5b5acff2e588]},
    Row{[__map-store.table_o_a0e591c7_2f01_4e8f_8cfe_8a2b064f7fdd]},
    Row{[__map-store.table_o_cdde3ccf_acb1_40ad_ac78_a6521e925426]},
    Row{[__map-store.table_o_dd915599_4d5e_4557_a30f_b7f5cacc7222]}]
not to contain
  [Row{[__map-store.table_o_741c4c93_cdc2_49ae_ba87_d500d3945898]}]
but found
  [Row{[__map-store.table_o_741c4c93_cdc2_49ae_ba87_d500d3945898]}]

	at com.hazelcast.jet.sql.SqlTestSupport.assertDoesNotContainRow(SqlTestSupport.java:349)
	at com.hazelcast.mapstore.GenericMapStoreIntegrationTest.lambda$testDestroy$1(GenericMapStoreIntegrationTest.java:329)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1277)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1296)
	at com.hazelcast.mapstore.GenericMapStoreIntegrationTest.testDestroy(GenericMapStoreIntegrationTest.java:328)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:750)

```
</details>


<details><summary>Standard output:</summary>

```
Finished Running Test: testGet in 0.107 seconds.
Started Running Test: testDestroy
10:09:31,490 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_a0e591c7_2f01_4e8f_8cfe_8a2b064f7fdd
10:09:31,495 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-2 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_741c4c93_cdc2_49ae_ba87_d500d3945898
10:09:31,495  INFO || - [DefaultRecordStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.partition-operation.thread-0 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Starting to load all keys for map table_o_741c4c93_cdc2_49ae_ba87_d500d3945898 on partitionId=6
10:09:31,508  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:31,511  WARN || - [SubmitJobOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:31,511  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-1 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:31,515 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-10 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_a0e591c7_2f01_4e8f_8cfe_8a2b064f7fdd
10:09:31,516 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_741c4c93_cdc2_49ae_ba87_d500d3945898
10:09:31,526 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-10 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Reading existing mapping for maptable_o_a0e591c7_2f01_4e8f_8cfe_8a2b064f7fdd
10:09:31,527  INFO || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-15 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Dropping mapping __map-store.table_o_741c4c93_cdc2_49ae_ba87_d500d3945898
10:09:31,527 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Reading existing mapping for maptable_o_741c4c93_cdc2_49ae_ba87_d500d3945898
10:09:31,528  INFO || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Dropping mapping __map-store.table_o_741c4c93_cdc2_49ae_ba87_d500d3945898
10:09:31,532  WARN || - [InitExecutionOperation] ForkJoinPool.commonPool-worker-0 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:31,532  WARN || - [SubmitJobOperation] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-10 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:31,533 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-18 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_741c4c93_cdc2_49ae_ba87_d500d3945898
10:09:31,533 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-6 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_741c4c93_cdc2_49ae_ba87_d500d3945898
10:09:31,547  WARN || - [InitExecutionOperation] ForkJoinPool.commonPool-worker-6 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:31,556 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-18 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Reading existing mapping for maptable_o_741c4c93_cdc2_49ae_ba87_d500d3945898
10:09:31,560  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-5 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:31,562  WARN || - [SubmitJobOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-9 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:33,685 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-12 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_096cc44c_fb22_42b1_a80b_e864148776a0
10:09:33,721  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-2 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:33,723  WARN || - [SubmitJobOperation] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-10 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:33,726 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-5 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_096cc44c_fb22_42b1_a80b_e864148776a0
10:09:33,738 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-5 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Reading existing mapping for maptable_o_096cc44c_fb22_42b1_a80b_e864148776a0
10:09:33,747  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-17 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:33,748  WARN || - [SubmitJobOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-11 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:33,863 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_dd915599_4d5e_4557_a30f_b7f5cacc7222
10:09:33,890  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-5 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:33,891  WARN || - [SubmitJobOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-5 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:33,896 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-15 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_dd915599_4d5e_4557_a30f_b7f5cacc7222
10:09:33,908 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-15 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Reading existing mapping for maptable_o_dd915599_4d5e_4557_a30f_b7f5cacc7222
10:09:33,916  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-16 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:33,917  WARN || - [SubmitJobOperation] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-11 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:33,990 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-5 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_94f6e2d4_1b39_4f98_8d4c_5b5acff2e588
10:09:34,007  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-11 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:34,015 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-4 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_94f6e2d4_1b39_4f98_8d4c_5b5acff2e588
10:09:34,026 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-4 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Reading existing mapping for maptable_o_94f6e2d4_1b39_4f98_8d4c_5b5acff2e588
10:09:34,034  WARN || - [InitExecutionOperation] ForkJoinPool.commonPool-worker-4 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:34,034  WARN || - [SubmitJobOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-17 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:34,106 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-15 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_cdde3ccf_acb1_40ad_ac78_a6521e925426
10:09:34,124  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-6 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:34,133 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_cdde3ccf_acb1_40ad_ac78_a6521e925426
10:09:34,144 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Reading existing mapping for maptable_o_cdde3ccf_acb1_40ad_ac78_a6521e925426
10:09:34,150  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:34,152  WARN || - [SubmitJobOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:36,397 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-20 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_68d997fe_d546_45fb_8651_8a2b08315b5f
10:09:36,422  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-5 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:36,423  WARN || - [SubmitJobOperation] hz.MySQLGenericMapStoreIntegrationTest_wonderful_bhabha.cached.thread-9 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:36,428 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-3 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Initializing for map table_o_68d997fe_d546_45fb_8651_8a2b08315b5f
10:09:36,440 DEBUG || - [GenericMapStore] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-3 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] Reading existing mapping for maptable_o_68d997fe_d546_45fb_8651_8a2b08315b5f
10:09:36,449  WARN || - [InitExecutionOperation] hz.MySQLGenericMapStoreIntegrationTest_fervent_bhabha.cached.thread-2 - [127.0.0.1]:5702 [dev] [5.3.0-SNAPSHOT] null
java.util.concurrent.CancellationException: null
	at com.hazelcast.jet.sql.impl.processors.RootResultConsumerSink.tryProcess(RootResultConsumerSink.java:97) ~[hazelcast-sql-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.lambda$stateMachineStep$2(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.util.Util.doWithClassLoader(Util.java:547) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:334) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:328) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:291) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:404) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:895) ~[?:1.8.0_351]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:369) ~[hazelcast-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_351]
10:09:37,303  INFO |testDestroy| - [SqlTestSupport] Time-limited test - Removing 0 cached plans in SqlTestSupport.@After
10:09:37,303  INFO |testDestroy| - [SqlTestSupport] Time-limited test - Removing 0 cached plans in SqlTestSupport.@After
10:09:37,304  INFO |testDestroy| - [SimpleTestInClusterSupport] Time-limited test - Ditching 0 jobs in SimpleTestInClusterSupport.@After: []
10:09:37,305  INFO |testDestroy| - [SimpleTestInClusterSupport] Time-limited test - Destroying 1 distributed objects in SimpleTestInClusterSupport.@After: [hz:impl:mapService/__sql.catalog]
BuildInfo right after testDestroy(com.hazelcast.mapstore.mysql.MySQLGenericMapStoreIntegrationTest): BuildInfo{version='5.3.0-SNAPSHOT', build='20230225', buildNumber=20230225, revision=6609411, enterprise=false, serializationVersion=1}
Hiccups measured while running test 'testDestroy(com.hazelcast.mapstore.mysql.MySQLGenericMapStoreIntegrationTest):'
10:09:30, accumulated pauses: 50 ms, max pause: 0 ms, pauses over 1000 ms: 0
10:09:35, accumulated pauses: 656 ms, max pause: 638 ms, pauses over 1000 ms: 0


No metrics recorded during the test

```
</details>

Standard output can be found here - https://s3.console.aws.amazon.com/s3/buckets/j-artifacts/Hazelcast-master-little-endian/242/

--- a/extensions/mapstore/src/test/java/com/hazelcast/mapstore/GenericMapStoreIntegrationTest.java
+++ b/extensions/mapstore/src/test/java/com/hazelcast/mapstore/GenericMapStoreIntegrationTest.java
@@ -345,7 +345,7 @@ public class GenericMapStoreIntegrationTest extends JdbcSqlTestSupport {
 
         Row row = new Row("__map-store." + tableName);
         List<Row> rows = Collections.singletonList(row);
-        assertTrueEventually(() -> assertDoesNotContainRow(client, "SHOW MAPPINGS", rows), 5);
+        assertTrueEventually(() -> assertDoesNotContainRow(client, "SHOW MAPPINGS", rows), 30);
     }
 
     @Test

